bgpd/bgp_vty.c

```diff
@@ -12815,9 +12817,13 @@ static void bgp_show_peer(struct vty *vty, struct peer *p, bool use_json,
 			"  Hold time is %d, keepalive interval is %d seconds\n",
 			p->v_holdtime, p->v_keepalive);
 		vty_out(vty, "  Configured hold time is %d",
```

Comment:
This is not a part of this change, but IMO we should tell the operator that it's seconds :smile: (keepalive already does it)